libs/langchain/tests/integration_tests/llms/test_vertexai.py

```diff
@@ -97,3 +100,26 @@ async def test_model_garden_agenerate() -> None:
     output = await llm.agenerate(["What is the meaning of life?", "How much is 2+2"])
     assert isinstance(output, LLMResult)
     assert len(output.generations) == 2
+
+
+def test_vertex_call_trigger_count_tokens(mocker) -> None:
```

Comment:
nits: I'm not sure this qualifies as an integration test. Probably, for a integration test we should rather actually invoke a call to VertexAI and check that it returns results successfully, wdyt?

Reply:
Hi @lkuligin:
For this testcase (test_vertex_call_trigger_count_tokens), i modify it, so it will call the underlying function directory, I also create the other testcase called "test_get_num_tokens_be_called_when_using_mapreduce_chain", because the token usage didn't be included from chain output. So it is hard to me to make sure if this function(count_tokens) will be called when user are using mapreduce method or not. This is why i decide to use marker to test this logic.

Please let me know your thoughts, or any suggestions i can make it more better. :)